src/metabase/models/collection/graph.clj

```diff
@@ -83,6 +83,15 @@
                                                               (group-id->perms group-id)
                                                               collection-ids)}))})))
 
+(defn- modify-instance-analytics-for-admins
+  "In the graph, override the instance analytics collection within the admin group to read."
+  [graph]
+  (let [admin-group-id      (:id (perms-group/admin))
+        audit-collection-id (t2/select-one-fn :id :model/Collection :entity_id (perms/default-audit-collection-entity-id))]
```

Comment:
Here's an idea: since the audit collection is static, why don't we memoize it in a similar way to the admin group in `metabase.models.permissions-group`? 

It should also probably be in an enterprise namespace somewhere... maybe `metabase-enterprise.audit-db` or a new one dedicated to collection stuff